src/util/native_fs_utils.js

```diff
@@ -419,6 +416,20 @@ async function get_user_by_distinguished_name({ distinguished_name }) {
     }
 }
 
+async function get_anonymous_account_by_name(anony_account_name) {
+    try {
+        if (!anony_account_name) throw new Error('no anonymous account name');
+        const context = get_process_fs_context();
+        const iam_path = path.join(config.NSFS_NC_CONF_DIR, 'accounts', anony_account_name + '.json');
```

Comment:
This function can do some re-uses :) 
1.  call get_config_file_path(config_type_path, file_name) for getting the file path
2. instead of 'accounts' use manage_nsfs_constants.CONFIG_SUBDIRS.ACCOUNTS
3. call get_config_data() for reading and parsing

Reply:
also, I prefer keeping the anon shortname as in object_sdk if you don't mind (instead of anony)